### PR TITLE
DEVDOCS-5605 [revise]: Headless, add callout for Catalyst

### PR DESCRIPTION
**Pull request by bc-andreadao**
# [DEVDOCS-5605]

## What changed?
- Added callout for Catalyst 

[DEVDOCS-5605]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-5605?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/api-docs/building-storefronts/nextjs-bigcommerce.mdx [hunk 1]
```diff
@@ -31,6 +31,11 @@ Next.js + BigCommerce requires a [BigCommerce sandbox](/docs/start/about/sandbox
 
 To get started, visit the [Next.js + BigCommerce repository (GitHub)](https://github.com/bigcommerce/nextjs-commerce). Use the [README](https://github.com/bigcommerce/nextjs-commerce/blob/main/README.md) and [Example environment variables file](https://github.com/bigcommerce/nextjs-commerce/blob/main/.env.example) to get oriented. You can use the [Deploy with Vercel button (GitHub)](https://github.com/bigcommerce/nextjs-commerce) at the top of the README to configure Vercel environment variables and deploy your storefront, then use the [Vercel CLI (Vercel)](https://vercel.com/docs/cli) to develop locally and see your changes in deployment at will.
 
+<Callout>
+**Looking for more?** <br/>
```
Comment (bc-traciporter) [inline docs/api-docs/building-storefronts/nextjs-bigcommerce.mdx:35]:
```suggestion
<br>**Looking for more?** <br/>
```

**Review by bc-traciporter — COMMENTED** [review]
I found one thing.

File: docs/api-docs/building-storefronts/nextjs-bigcommerce.mdx (hunk 1 above)
Comment (bc-andreadao) [reply docs/api-docs/building-storefronts/nextjs-bigcommerce.mdx:35]: @bc-traciporter You don't need that opening tag. Right now, it appears like this in the preview:
callout

**Review by bc-traciporter — APPROVED** [review]
LGTM!

**Pull request merged**